content/courses/index.njk

```diff
@@ -0,0 +1,16 @@
+---
+title: Courses
+index: Course
+list_label: All Courses
+sub: Frontend web development courses for curious developers
+summary: |
+  Video and email courses for frontend web developers on modern CSS including
+  anchor positioning, grid & subgrid, container queries and more!
+---
```

Comment:
@jgerigmeyer when I moved the contents of this folder up a level, the email course no longer is displayed on this `courses` page. Should it be nested in an email folder or something like the video course?

Reply:
@stacyk Try again? I think I fixed it.

Reply:
Thank you! That works.